package.json

```diff
@@ -0,0 +1,131 @@
+{
+  "name": "hipo-frontend-ts-boilerplate",
+  "version": "0.0.0",
+  "description": "Hipo Frontend TypeScript Boilerplate",
+  "private": true,
+  "scripts": {
+    "build:release": "webpack --config webpack.release.config.js  --progress --colors",
+    "build:dev": "webpack --config webpack.dev.config.js --progress --colors",
+    "build:dev:live": "./node_modules/.bin/webpack-dev-server --config webpack.dev.config.js --progress --colors --watch",
+    "start": "npm run build:dev:live",
+    "type-check": "echo \"type-checking...\" && tsc --noEmit",
+    "type-check:watch": "npm run type-check -- --watch",
+    "eslint": "./node_modules/.bin/eslint --ext .jsx,.js,.tsx,.ts src/",
+    "eslint:fix": "./node_modules/.bin/eslint --fix --ext .jsx,.js,.tsx,.ts src/",
+    "stylelint": "echo \"stylelint-checking...\" && ./node_modules/.bin/stylelint 'src/**/*.{css,scss}'",
+    "stylelint:fix": "npm run stylelint -- --fix",
+    "prettify": "./node_modules/.bin/prettier --config ./.prettierrc.js --write 'src/**/*.{ts,tsx}'",
+    "lint": "npm run stylelint; npm run eslint",
+    "test": "./node_modules/.bin/jest",
+    "test:watch": "npm t -- --watch",
+    "changelog": "./node_modules/.bin/conventional-changelog --config ./node_modules/hipo-web-conventional-changelog/index.js --context ./changelog-context-variables.json -o CHANGELOG.md -r 0",
+    "checkPackageUpdates": "npx npm-check-updates"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Hipo/hipo-frontend-ts-boilerplate.git"
```

Comment:
Could this be old name of this repo?

Reply:
I don't think thats important as this is just a template and that you would change these anyway for a real project